### PR TITLE
Task/aasdk 175 common banner updates part 3

### PR DESCRIPTION
**Pull request by gunhansancar**
## JIRA
[AASDK-175]

## DESCRIPTION
Updates video tests for the common module

## SCREENSHOTS
n/a

[AASDK-175]: https://superawesomeltd.atlassian.net/browse/AASDK-175?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: superawesome-common/src/main/java/tv/superawesome/sdk/publisher/common/ui/video/VideoEvents.kt [hunk 1]
```diff
@@ -49,9 +50,12 @@ internal class VideoEvents(
     }
 
     public fun time(videoPlayer: IVideoPlayer?, time: Int, duration: Int) {
+        Log.i("gunhanx","time update: time: $time duration: $duration")
         if (viewableDetector?.isVisible == null) {
             viewableDetector?.isVisible = {
                 scope.launch {
+                    Log.i("gunhanx","time update: time: $time oneSecondDwellTime")
```
Comment (me2k) [inline superawesome-common/src/main/java/tv/superawesome/sdk/publisher/common/ui/video/VideoEvents.kt:57]: Is this needed?

File: superawesome-common/src/main/java/tv/superawesome/sdk/publisher/common/ui/video/VideoEvents.kt [hunk 2]
```diff
@@ -49,9 +50,12 @@ internal class VideoEvents(
     }
 
     public fun time(videoPlayer: IVideoPlayer?, time: Int, duration: Int) {
+        Log.i("gunhanx","time update: time: $time duration: $duration")
```
Comment (me2k) [inline superawesome-common/src/main/java/tv/superawesome/sdk/publisher/common/ui/video/VideoEvents.kt:53]: Is this needed?

**Review by me2k — APPROVED** [review]
Approved, but see my comment about a random log statement

File: superawesome-common/src/main/java/tv/superawesome/sdk/publisher/common/ui/video/VideoEvents.kt (hunk 1 above)
Comment (gunhansancar) [reply superawesome-common/src/main/java/tv/superawesome/sdk/publisher/common/ui/video/VideoEvents.kt:57]: Let me clean up

File: superawesome-common/src/main/java/tv/superawesome/sdk/publisher/common/ui/video/VideoEvents.kt (hunk 2 above)
Comment (gunhansancar) [reply superawesome-common/src/main/java/tv/superawesome/sdk/publisher/common/ui/video/VideoEvents.kt:53]: Let me clean up

**Pull request merged**